### ISSUE TITLE
Fix screen tearing

### ISSUE THREAD

**Issue by Srykah** [OpMonTeam/OpMon-Archive#111]
I have felt quite a large amount of screen tearing when I played the game, especially when moving up or down in the world map. Screen tearing is usually removed by enbling VSync, but as I saw it in the code, a fixed frame frequency of 30 fps is used for rendering as well as for computing game objects' speeds. I think you should dissociate the game update logic and the rendering, so to let users choose if they want to play with VSync on or with maximum FPS.

**Comment by milkyroute** [comment]
I agree with you, this system is a mistake I've done quite a long time ago. I should change it as you say, but I always delay it. So I think I will do it for the 0.17, but I promise nothing. Except if somebody want to do it before.

**Comment by Srykah** [comment]
I can do it as soon as I am over cleaning up the rest of the code ^^
